src/hash.jl

```diff
@@ -1,5 +1,4 @@
-export hash
+using MurmurHash3: mmhash128_a
 
-import Base.hash
-
-Base.hash(x::ShortString, h::UInt) = hash(String(x), h)
+Base.hash(x::ShortString, h::UInt) =
+    last(mmhash128_a(sizeof(x), ntoh(x.size_content), h%UInt32)) + h
```

Comment:
```suggestion
    last(mmhash128_a(sizeof(x), bswap(x.size_content), h%UInt32)) + h
```

Reply:
Sure, what's the difference?

Reply:
ntoh is short for network to host, so if host (the computer executing it) is little endian or big little endian it does it differently right? So `bswap` will always swap them? I am bit confused. here.

Reply:
`bswap` will always swap.
`nthoh` will not swap if julia is running on a big endian machine.
Only if it is on a little endian machine.
Julia only runs on little endian machines however.
Main advantage is that `bswap` says what it does without remembering  network to host.